dockstore-client/src/test/java/io/dockstore/client/cli/WorkflowInDirectoryTestIT.java

```diff
@@ -62,6 +63,18 @@ public void testWorkflowRunInDirectory() {
         this.baseWorkflowTest(cwlFile, cwlJSON, false, "workflow");
     }
 
+    /**
+     * This tests whether cwltool can execute a workflow that contains an empty array hints property
+     * cwltool 1.0.20190621234233 does not seem able to do this anymore
+     */
+    @Ignore
```

Comment:
You can actually do something like @Ignore(" This tests whether cwltool can execute a workflow that contains an empty array hints property ...")